sklearn/utils/estimator_checks.py

```diff
@@ -3648,3 +3648,119 @@ def check_estimator_get_tags_default_keys(name, estimator_orig):
         f"{name}._get_tags() is missing entries for the following default tags"
         f": {default_tags_keys - tags_keys.intersection(default_tags_keys)}"
     )
+
+
+def check_dataframe_column_names_consistency(name, estimator_orig):
+    try:
+        import pandas as pd
+    except ImportError:
```

Comment:
nit
```python
pd = pytest.importorskip("pandas", reason=""pandas is not installed: not checking column name consistency for pandas")
```

Reply:
We can not use `pytest` in `estimator_checks`.